NotificationApi/NotificationApi/Services/CreateNotificationService.cs

```diff
@@ -29,8 +37,28 @@ public async Task CreateEmailNotificationAsync(CreateEmailNotificationCommand no
 
             await _commandHandler.Handle(notificationCommand);
             var requestParameters = parameters.ToDictionary(x => x.Key, x => (dynamic)x.Value);
-            var emailNotificationResponse = await _asyncNotificationClient.SendEmailAsync(notificationCommand.ContactEmail, template.NotifyTemplateId.ToString(), requestParameters, notificationCommand.NotificationId.ToString());
+
+            var emailNotificationResponse = await SendEmailAsyncRetry(notificationCommand.ContactEmail, template.NotifyTemplateId.ToString(), requestParameters, notificationCommand.NotificationId.ToString());
             await _commandHandler.Handle(new UpdateNotificationSentCommand(notificationCommand.NotificationId, emailNotificationResponse.id, emailNotificationResponse.content.body));
         }
+
+        public async Task<EmailNotificationResponse> SendEmailAsyncRetry(string contactEmail, string notifyTemplateId, Dictionary<string, dynamic> requestParameters, string notificationId)
+        {
+            var maxRetryAttempts = 2;
+            var pauseBetweenFailures = TimeSpan.FromSeconds(5);
+
+            var result = await _pollyRetryService.WaitAndRetryAsync<Exception, EmailNotificationResponse>
+           (
+               maxRetryAttempts,
+               _ => pauseBetweenFailures,
```

Comment:
I had some concerns over the unused parameter here potentially causing confusion but this is the way it is implemented in VideoApi and a similar example here https://alastaircrabtree.com/implementing-the-retry-pattern-using-polly/ 

Happy to approve with that in mind.